docs/UserGuide.adoc

```diff
@@ -207,41 +207,64 @@ exp/23-Nov-2020 a/21`
 
-For example, when a user wants to add a patient, the display box will prompt the user to enter the patient's name.
-For the user to add a patient that is named "John Doe", type `John Doe` and then press enter.
-After entering the name, the patient's name will be displayed in the form, as shown below.
+That is all the important commands for this feature, now let's see an example on how the feature will look like.
+When you want to add a `Patient`, type `add t/patient` on the `Command Box` and a form will appear as below and the `Result Box` will prompt the user to enter the patient's name.
 
-image::nameField.png[width="790"]
+image::addForm.png[width="790"]
 
-Inputs for each attribute will be checked, if it is a valid input then the user input will automatically reflected
-on the form, if it is not valid, ORGANice will show a warning to the user.
+If you want to add a patient named "John Doe", type `John Doe` and then press enter.
+After entering the name, the patient's name will be displayed on the form as seen below and the progress bar will increase indicating the your progress in filling up the form.
 
-The following screenshot shows the warning that ORGANice will give to the user if the input is not valid:
+image::nameFilled.png[width="790"]
 
-image::wrongInput.png[width="790"]
+Input for each attribute will be checked, if it is a valid input then the user input will automatically be reflected
+on the form, if it is not valid, ORGANice will show a warning to you.
 
-After all of the attributes have been specified, the user can type `/done` to add the person to the ORGANice list of persons
-or `/abort` if the user decided not to add the person into the list.
+The following screenshot shows a warning that ORGANice will give to you if your input is not valid:
 
-image::allField.png[width="790"]
+image::invalidInput.png[width="790"]
 
-When the user typed `/done` the UI will changed back to it's original state and shows a success message suggesting that
-the user has successfully added a new person into the list.
+After you have entered a particular attribute and realized that you have made a mistake you can type in `/undo` to remove the last entry you entered.
+In the example below, a user might have entered a wrong `Patient's` name and thus typed in `/undo` and the name that is previously reflected on the form is removed.
+
+image::undoCommand.png[width="790"]
+
+After all of the attributes have been specified, you can type `/done` to add the person to the ORGANice list of persons,
+`/abort` if you decided not to add the person into the list, or click the `Find Match` button to directly add the person and
+find the potential donors at the same time.
+Note that the `Find Match` button will only appear on the `Patient` form.
+
+image::allFieldSpecified.png[width="790"]
+
+When you typed in `/done` the UI will changed back to it's original state and shows a success message suggesting that
```

Comment:
When you type in `/done`, the Person List will return to its original state of displaying people in ORGANice. 
The Message Display Box will show a success message, indicating that you have successfully added a new person to the Person List.